.vscode/settings.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Minor, but this file (or indeed, the whole `.vscode` folder) is probably best to be gitignored.